<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewEnv</class>
 <widget class="QWidget" name="NewEnv">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>498</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>名称</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="name"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QRadioButton" name="radioButton">
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>新的虚拟环境</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>位置</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="path"/>
       </item>
       <item>
        <widget class="QToolButton" name="path_btn">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,9">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>基于</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="base_py_list"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QRadioButton" name="radioButton_2">
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>选取外部环境</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,9,0">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>位置</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="exis_path">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="exit_path_btn">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="ok_btn">
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancel_btn">
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
